src/components/NewService.vue

```diff
@@ -122,14 +122,15 @@
       </p>
       <span class="select">
         <select v-model="configuration[linkedService.as]">
-          <option v-for="service in services" v-bind:value="service[linkedService.retrieve]">
+          <option v-for="service in filterServicesByType(linkedService.service_type)" v-bind:value="service[linkedService.retrieve]">
```

Comment:
Where's the definition of the linkedService again? Which attributes does it have?

From what I read in the code, it's just a service object, right?. If so, don't you have access to the service_type_id (as per new line 215)? I'm asking all of this to see if you could use findServiceTypeById and get rid of the double filtering done below.

Reply:
@taliaga linkedService refers to the linkable services described in the configuration templates.

And I can't avoid the double filtering because service and service type are different models that refer to each other by id.

Reply:
>@taliaga linkedService refers to the linkable services described in the configuration templates.

Do you have a pointer to help me refresh my memory of that definition of a linkedService model?

> And I can't avoid the double filtering because service and service type are different models that refer to each other by id.

Sure that.